packages/nginx/files/nginx-proxy-gen

```diff
@@ -5,13 +5,18 @@
 # SPDX-License-Identifier: GPL-2.0-only
 #
 
+"""
+This script generates location entries for nginx, find more info in the README.md file.
+"""
+
 import os
-import sys
 from glob import glob
+
 from euci import EUci
 
 u = EUci()
 
+
```

Comment:
Automatic prettier, I hate you! :D

Please avoid unnecessary changes: the patch it's already quite big

Reply:
Whops, my bad :sweat: